sumi/collective_message.cc

```diff
@@ -92,7 +92,7 @@ CollectiveWorkMessage::toString() const
   return sprockit::sprintf(
     "message for collective %s recver=%d sender=%d nbytes=%d round=%d tag=%d %s %d->%d",
      Collective::tostr(type_), recver(), sender(), payloadBytes(), round_, tag_,
-     sstmac::hw::NetworkMessage::typeStr(), toaddr(), fromaddr());
+     sstmac::hw::NetworkMessage::typeStr(), fromaddr(), toaddr());
```

Comment:
Was this a bug in the old code?

Reply:
Depends on what you think is a bug. recver() and sender() don't change when a message is "reversed" but toaddr() and fromaddr() do. Jeremy must have liked it the way it was, I want to see arrow from->to so I can make sense of it. Just a convention for debug output, really.

Reply:
Seems fine